Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/common/SparkBatchJob.java

```diff
@@ -1102,19 +1120,29 @@ private void setDriverLogConversionMode(@Nullable DriverLogConversionMode driver
     @Override
     public Observable<? extends ISparkBatchJob> deploy(@NotNull String artifactPath) {
         if (accessToken != null) {
-            return JobUtils.deployArtifactToADLS(artifactPath, adlRootPath, accessToken)
+            //use ADLS GEN1
+            return JobUtils.deployArtifactToADLS(artifactPath, destinationRootPath, accessToken)
                     .map(path -> {
                         getSubmissionParameter().setFilePath(path);
                         return this;
                     });
+        } else if (destinationRootPath != null) {
```

Comment:
Try to use schem, such as `http:// https:// webhdfs://` to distinguish the ADLS v1 and webhdfs